<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFindDlg</class>
 <widget class="QDialog" name="QFindDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>82</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查找</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>查找内容:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditContent"/>
   </item>
   <item row="0" column="2" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonFindNext">
       <property name="text">
        <string>查找下一个</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="checkBoxCaseSensitive">
     <property name="text">
      <string>区分大小写</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>方向</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radioButtonUp">
        <property name="text">
         <string>向上</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonDown">
        <property name="text">
         <string>向下</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="checkBoxWholeWord">
     <property name="text">
      <string>全字匹配</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>QFindDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
